scripts/Muon/GUI/Common/plot_widget/plotting_canvas/plot_color_queue.py

```diff
@@ -0,0 +1,37 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+import heapq
```

Comment:
can we have a comment with a link to the doc for this? I dont think it is common knowledge